flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/internal/converter/OracleRowConverter.java

```diff
@@ -33,6 +52,103 @@ public OracleRowConverter(RowType rowType) {
         super(rowType);
     }
 
+    @Override
+    public JdbcDeserializationConverter createInternalConverter(LogicalType type) {
+        // TODO: please consider all Oracle Data Types like oracle.sql.OracleBlob,
```

Comment:
@eskabetxe I didn't finish all types here, I just made a basic draft.
and I think we need support all oracle data type and update the datatype mapping as well